data/json/items/tool_armor.json

```diff
@@ -60,6 +60,44 @@
     "techniques": [ "WBLOCK_1" ],
     "magazine_well": 1
   },
+  {
+    "id": "welding_mask",
+    "type": "ARMOR",
+    "name": "welding mask",
+    "description": "A steel mask with a very dark visor, meant to protect the entire face while welding.  It also serves as good armor, but makes it hard to see.  Activate it to push it up your head.",
```

Comment:
This needs to count as a blindfold, not just make it hard to see.

Reply:
Should welding goggles count as blindfold too, or welding mask visors are much darker than that? Just to keep things consistent.

Reply:
Typically they're notably darker than welding goggles, which are intended more for stuff like chemical torch welding and cutting.

Reply:
Are there actually any steel welding masks commercially available? I'm having trouble finding them, and difficulty believing a welder would want to wear something so heavy when there are lighter solutions.

Reply:
Well, what material are they usually made of? I might change it to that then.

Reply:
Plastic.

Reply:
Done, changed the material.